package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@square/maker",
-  "version": "1.2.2",
```

Comment:
Since this is a new feature / backwards-compatible additive change you should increment the minor version instead of the patch version, e.g. `1.3.0` instead of `1.2.3`.